.gitignore

```diff
@@ -22,4 +22,21 @@ specification_compliance_report.html
 */**/.idea/
 /compliance
 /.smithy.lsp.log
-*/**/Folder.DotSettings.user
\ No newline at end of file
+*/**/Folder.DotSettings.user
+
+# These files are not tracked.
+# The release will force-commit these files, then the post-release commit will force-remove them.
+# Scripts at /scripts/release will run on all files between START_RELEASE_IGNORED_FILES and END_RELEASE_IGNORED_FILES.
+# START_RELEASE_IGNORED_FILES
+**/*py.dtr
+# Ignore everything in ImplementationFromDafny-go...
+**/ImplementationFromDafny-go/**
+# ... except for files containing "smithygenerated"...
+!**/ImplementationFromDafny-go/**/*smithygenerated*
+# ... except for files containing "extern" ...
+!**/ImplementationFromDafny-go/**/*extern*
+# ... except for go.mod ...
+!**/ImplementationFromDafny-go/**/go.mod
```

Comment:
For Kms and Ddb, we also generate `shim.go` . But not sure if we want include the shims for other projects (I think we should).

Reply:
Do you mean the wrapped shims for testing local services? I think we commit those. (But we haven't released one yet, since the only place we've committed this is in MPL test vectors, which we've never released.)
If those are also just called `shim.go`, I'll add `shim.go` to this section